notebooks/Testgrid_flakiness_detection.ipynb

```diff
@@ -0,0 +1,2907 @@
+{
```

Comment:
<p><code>s/thersold/threshold/</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
The param in `calc_optimal_flakiness_score` function still contains this typo. It's a nit pick though.

Reply:
^^ @SankBad

Reply:
@tumido , Earlier I didn't understand this comment. My bad :facepalm: 
Just fixed this.